RubberduckTests/Inspections/SheetAccessedUsingStringInspectionTests.cs

```diff
@@ -11,9 +11,11 @@
 
 namespace RubberduckTests.Inspections
 {
+    [Ignore("TODO: figure out how to mock retrieving the document module supertypes from the usercode ITypeLib; none of these tests are conclusive until then...")]
```

Comment:
Why not extract the current logic in the resolve runner base into an `ISuperTypeNamesForDocumentsProvider` with a single method taking a project id and a collection of document declarations and returning a dictionary document module - > super type names? That could then be injected and mocked in the tests.

Reply:
On it.